requirements-test.txt

```diff
@@ -0,0 +1,2 @@
+-r requirements.txt
```

Comment:
I didn't know we could have a seperate requirements.txt for tests. Thanks a lot for sharing!

Reply:
Yeah :)

There are even alternative ways to manage dependencies.

- [pipenv](https://pipenv.pypa.io/en/latest/)
- [poetry](https://python-poetry.org/)